Jenkinsfile.aws-test

```diff
@@ -0,0 +1,74 @@
+def NODE = "rhcos-jenkins"
+def AWS_REGION = "us-east-1"
+
+// this var conveniently refers to a location on the server as well as the
+// local dir we sync to/from
+def images = "/srv/rhcos/output/images"
+
+node(NODE) {
+    stage("Clean workspace") {
+       step([$class: 'WsCleanup'])
+    }
+    checkout scm
+    utils = load("pipeline-utils.groovy")
+
+    // This job should only be triggered by Jenkinsfile.cloud
+    utils.define_properties(null)
+
+    if (params.DRY_RUN) {
+        echo "DRY_RUN set, skipping aws tests and launch permissions"
+        currentBuild.result = 'NOT_BUILT'
+        currentBuild.description = '(dry run)'
+        return
+    }
+
+    utils.inside_assembler_container("-v /srv:/srv") {
+    stage("Sync In") {
+        withCredentials([
+            string(credentialsId: params.ARTIFACT_SERVER, variable: 'ARTIFACT_SERVER'),
+            sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
+        ]) {
+            sh "mkdir -p ${images}"
+            utils.rsync_dir_in(ARTIFACT_SERVER, KEY_FILE, images)
+        }
+    }
+
+    try {
+    stage("Run Kola tests on intermediate aws image") {
+            withCredentials([
+                [$class: 'AmazonWebServicesCredentialsBinding', credentialsId: params.AWS_CREDENTIALS],
+                string(credentialsId: params.S3_PRIVATE_BUCKET, variable: 'S3_PRIVATE_BUCKET'),
+                string(credentialsId: params.AWS_CI_ACCOUNT, variable: 'AWS_CI_ACCOUNT'),
+            ]) {
+                currentBuild.description = "ami: ${ami_intermediate}"
+                sh """
+                    # Do testing with intermediate aws image passed in by cloud job
+                    if ! kola -b rhcos -p aws --aws-type ${aws_type} --tapfile rhcos-aws.tap --aws-ami ${ami_intermediate} --aws-region ${AWS_REGION} run; then
+                        exit 1
+                    fi
+
+                    # Tests pass, tag the json in the artifact server
+                    # and give launch permissions to OpenShift CI
+                    cp ${dirpath}/aws-${AWS_REGION}-smoketested.json ${dirpath}/aws-${AWS_REGION}-awstested.json
```

Comment:
The problem is that `-awstested.json` will go away fairly quickly as the cloud images churn.

So anyone who wants to find out "what is the latest AMI that passed basic testing" is going to be fighting `404s`.

You know I keep framing every problem we have as "we need tagging" :smile: 

Anyways...a tactical fix here might be to stick `-awstested.json` *outside* of the versioned image directory, i.e. just sync to `${images}/aws-${AWS_REGION}-tested.json`.

Reply:
:+1: on keeping the latest successfully tested image around until it's replaced.